test/webgl-utils/custom-parameter-settings.js

```diff
@@ -59,7 +59,7 @@ export const GL_PARAMETER_SETTINGS_ONE = {
   // WEBGL2 / EXTENSIONS
   // gl1: 'OES_standard_derivatives'
   [GL.FRAGMENT_SHADER_DERIVATIVE_HINT]: GL.FASTEST,
-  // RASTERIZER_DISCARD ...
```

Comment:
Maybe name this file `sample-parameters.js` (and `sample-settings.js` if splitting it, see below)?